go-packages/meep-net-char-mgr/algo-segment.go

```diff
@@ -403,7 +404,27 @@ func (algo *SegmentAlgorithm) populateFlow(flowName string, srcElement *SegAlgoN
 	flow.ConfiguredNetChar.Jitter = 0
 	flow.ConfiguredNetChar.PacketLoss = 0
 	// Create a new path for this flow
+	oldPath := flow.Path
 	flow.Path = algo.createPath(flowName, srcElement, destElement, model)
+	diffPath := algo.comparePath(oldPath, flow.Path)
```

Comment:
Can you assign directly to NewPath and remove diffPath?

Reply:
done.. got rid of diffPath